### DISCUSSION TITLE
What is the difference between conversation_id and parent_id? How to use them?

### DISCUSSION THREAD

**Discussion by Lukangkang123** [acheong08/ChatGPT#861] [Q&A]
I want to connect this project to QQ and set up different sessions for each QQ friend. I noticed that the "ask" method has two parameters :conversation_id and parent_id, which one should I choose, and how to use it?

**Comment by linhandev** [comment]
I think it will be clear after you go through the [ask() code](https://github.com/acheong08/ChatGPT/blob/main/src/revChatGPT/V1.py#L172). Seems to me conversation id points to a conversation, parent id points to the previous message
